ERROR - 2023-05-15 00:47:42 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'qlbaocao', 3306, '', 0)
#1 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
#5 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(NULL)
#6 D:\Xampp\htdocs\Codeigniter4\app\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
#7 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#9 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\Xampp\htdocs\Codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\Codeigniter4\system\Commands\Server\rewrite.php(47): require_once('D:\\Xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
#4 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(NULL)
#5 D:\Xampp\htdocs\Codeigniter4\app\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
#6 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#7 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#8 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\Xampp\htdocs\Codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\Codeigniter4\system\Commands\Server\rewrite.php(47): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-15 00:47:42 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_function`
WHERE `user_id` IS NULL', [], false)
 3 APPPATH\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(null)
 5 APPPATH\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-05-15 00:48:35 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'qlbaocao', 3306, '', 0)
#1 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
#5 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(NULL)
#6 D:\Xampp\htdocs\Codeigniter4\app\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
#7 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#9 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\Xampp\htdocs\Codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\Codeigniter4\system\Commands\Server\rewrite.php(47): require_once('D:\\Xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
#4 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(NULL)
#5 D:\Xampp\htdocs\Codeigniter4\app\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
#6 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#7 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#8 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\Xampp\htdocs\Codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\Codeigniter4\system\Commands\Server\rewrite.php(47): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-15 00:48:35 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_function`
WHERE `user_id` IS NULL', [], false)
 3 APPPATH\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(null)
 5 APPPATH\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-05-15 00:49:46 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'qlbaocao', 3306, '', 0)
#1 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
#5 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(NULL)
#6 D:\Xampp\htdocs\Codeigniter4\app\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
#7 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#9 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\Xampp\htdocs\Codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\Codeigniter4\system\Commands\Server\rewrite.php(47): require_once('D:\\Xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in D:\Xampp\htdocs\Codeigniter4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\Xampp\htdocs\Codeigniter4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
#4 D:\Xampp\htdocs\Codeigniter4\app\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(NULL)
#5 D:\Xampp\htdocs\Codeigniter4\app\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
#6 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#7 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#8 D:\Xampp\htdocs\Codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\Xampp\htdocs\Codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\Codeigniter4\system\Commands\Server\rewrite.php(47): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-15 00:49:46 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user_function`
WHERE `user_id` IS NULL', [], false)
 3 APPPATH\Libraries\lib_auth.php(38): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Libraries\lib_auth.php(13): App\Libraries\lib_auth->getFuntionUser(null)
 5 APPPATH\Controllers\BaseController.php(58): App\Libraries\lib_auth->__construct()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 00:58:51 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 00:59:17 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:00:13 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:00:59 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:01:13 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:01:34 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:02:21 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:03:54 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 159.
 1 APPPATH\Models\ReportGroupModel.php(159): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 159)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:05:24 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 158.
 1 APPPATH\Models\ReportGroupModel.php(158): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 158)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:05:42 --> Attempt to read property "row_sum" on array
in APPPATH\Models\ReportGroupModel.php on line 158.
 1 APPPATH\Models\ReportGroupModel.php(158): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_sum" on array', 'APPPATH\\Models\\ReportGroupModel.php', 158)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:07:50 --> Attempt to read property "row_id" on array
in APPPATH\Models\ReportGroupModel.php on line 161.
 1 APPPATH\Models\ReportGroupModel.php(161): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "row_id" on array', 'APPPATH\\Models\\ReportGroupModel.php', 161)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:10:19 --> Undefined array key "value1_total"
in APPPATH\Models\ReportGroupModel.php on line 172.
 1 APPPATH\Models\ReportGroupModel.php(172): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "value1_total"', 'APPPATH\\Models\\ReportGroupModel.php', 172)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-05-15 01:18:14 --> Undefined array key "value1_total"
in APPPATH\Models\ReportGroupModel.php on line 174.
 1 APPPATH\Models\ReportGroupModel.php(174): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "value1_total"', 'APPPATH\\Models\\ReportGroupModel.php', 174)
 2 APPPATH\Controllers\Dashboard\ReportGroupController.php(41): App\Models\ReportGroupModel->getListReportGroupPrint([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard\ReportGroupController->data_report_group_print()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard\ReportGroupController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
